backend/src/main/resources/db/migration/internal/V0.292__Update_vessels_table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE public.vessels
+    ALTER COLUMN declared_fishing_gears SET NOT NULL;
```

Comment:
Il y avait `has_esacapt` aussi, tu ne le mets pas finalement ?

Reply:
Colonne déjà not null finalement